data/json/items/book/misc.json

```
@@ -931,13 +931,105 @@
     "name": "book of philosophy",
     "name_plural": "books of philosophy",
     "description": "A deep discussion of morality with an emphasis on epistemology and logic.",
-    "weight": 1400,
+    "snippet_category": [
+      {
+        "id": "philosophy1",
+        "text": "This is a copy of Nietzsche's \"Beyond Good and Evil\". It's cover dog-eared and creased."
```

```suggestion
        "text": "This is a copy of Nietzsche's \"Beyond Good and Evil\". Its cover is dog-eared and creased."
```
I don't know how well 'dog-eared' translates (and I'm only guessing) - maybe 'worn and creased' ? 'faded and creased' ?

---

https://www.merriam-webster.com/dictionary/dog-eared
Isn't uncomon. MW is an american dict afaik, should be fine.